qualtran/drawing/bloq_counts_graph.py

```diff
@@ -241,6 +241,7 @@ def format_qec_gates_cost(cls, val: 'GateCounts', agg: Optional[str] = None) ->
             'and_bloq': 'Ands',
             'clifford': 'Cliffords',
             'rotation': 'Rotations',
+            'rotations': 'Rotations',
```

Comment:
rotation and rotations map to Rotations correct? There's no extra s on the previous Rotation

Reply:
this matches all the field names in the various "aggregated" formats. Since I added support for `TComplexity`, we need the with-an-s version as well since its `TComplexity.rotations` (although the other fields don't have the s 🤷‍♀️ )